tests/test_distribution.py

```diff
@@ -76,6 +90,126 @@ def test_properties(self):
         self.assertEqual(distribution.__str__(), expected_repr)
         self.assertEqual(distribution.__repr__(), expected_repr)
 
+    def test_dist_info_file(self):
+        """Test that method
+        :meth:`~pythonforandroid.distribution.Distribution.dist_info_file`
+        returns the expected value."""
+        self.setUp_distribution_with_bootstrap(
+            Bootstrap().get_bootstrap("sdl2", self.ctx)
+        )
+        distribution = self.ctx.bootstrap.distribution
+        self.assertEqual(
+            distribution.dist_info_file,
+            os.path.join(distribution.dist_dir, "dist_info.json")
+        )
+
+    @mock.patch("pythonforandroid.distribution.open", create=True)
+    def test_get_dist_info(self, mock_open):
+        """Test that method
+        :meth:`~pythonforandroid.distribution.Distribution.get_dist_info`
+        calls the proper methods and returns the proper value."""
+        self.setUp_distribution_with_bootstrap(
+            Bootstrap().get_bootstrap("sdl2", self.ctx)
+        )
+        mock_open.side_effect = [
+            mock.mock_open(read_data=json.dumps(dist_info_data)).return_value
+        ]
```

Comment:
Using mock open this way is somehow bugging me, I guess I'm not used to it.
How about something more conventional?
```python
      def test_get_dist_info(self):
          """Test that method
          :meth:`~pythonforandroid.distribution.Distribution.get_dist_info`
          calls the proper methods and returns the proper value."""
          self.setUp_distribution_with_bootstrap(
              Bootstrap().get_bootstrap("sdl2", self.ctx)
          )
          mock_open = mock.mock_open(read_data=json.dumps(dist_info_data))
          with mock.patch("pythonforandroid.distribution.open", mock_open):
              dist_info = self.ctx.bootstrap.distribution.get_dist_info("/fake_dir")
          mock_open.assert_called_once_with("/fake_dir/dist_info.json", "r")
          self.assertIsInstance(dist_info, dict)
```
It's really passing the mock_open.side_effect with the other `return_value` that is bugging me, it feels unnatural.